lib/datadog/tracing/contrib/karafka/monitor.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require_relative 'ext'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Karafka
+        # Custom monitor for Karafka
```

Comment:
```suggestion
        # Custom monitor for Karafka.
        # Creating a custom monitor, instead of subscribing to an event (e.g. `Karafka.monitor.subscribe 'worker.processed'`),
        # is required because event subscriptions cannot wrap the event execution (`yield`).
```